pkg/controller/bootstrap/bootstrap.go

```diff
@@ -182,6 +182,9 @@ func (b *Bootstrap) Run(destDir string) error {
 		}
 		configs = append(configs, kconfigs...)
 	}
+	if err := mcfgv1.ProxyInspectTest(cconfig.Spec.Proxy); err != nil {
```

Comment:
Below you check if proxy != nil but here you don't?

Reply:
proxy != nil this was checked inside the function body ProxyInspectTest